css/style.css

```diff
@@ -20,66 +20,265 @@ ul {
 .headline {
-  margin-top: 70px;
+  margin-top: 100px;
+  padding-top: 40px;
   color: #172b4d;
 }
 
 .headline h1 {
   font-weight: 700;
   font-size: 40px;
+  width: 80%;
 }
 
 .headline p {
   margin-top: 20px;
   font-weight: 400;
-  font-size: 16px;
   color: #344563;
 }
 
-.headline span {
+.lets-connect {
   display: block;
   margin-top: 10px;
   font-weight: 500;
   font-size: 16px;
   color: #7f8cff;
 }
 
-.headline .social-icons {
+.social-icons {
   margin-top: 10px;
   display: flex;
 }
 
-.headline .social-icons a {
+.social-icons li a {
   color: #505f79;
   font-size: 25px;
   margin-right: 10px;
 }
+
+/* Main Styles */
+main {
+  margin: 40px 0;
+  min-height: 100vh;
+  width: 100vw;
+}
+
+/* Work Section styles */
+.work-section {
+  display: grid;
+  grid-template-areas:
+    'card-1 card-1 card-1 card-1'
+    'card-2 card-2 card-2 card-2'
+    'card-3 card-3 card-3 card-3'
+    'card-4 card-4 card-4 card-4';
+  gap: 80px;
+  width: 100%;
+  padding: 140px 20px 140px 20px;
+  background: #e5e5e5;
+}
+
+.work-card {
+  display: flex;
+  flex-direction: column;
+  padding: 16px;
+  gap: 12px;
+  width: 327px;
+  margin: auto;
+  min-height: 554px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+}
+
+.card-1 {
+  grid-area: card-1;
+}
+
+.card-2 {
+  grid-area: card-2;
+  height: 592px;
+}
+
+.card-3 {
+  grid-area: card-3;
+}
+
+.card-4 {
+  grid-area: card-4;
+  height: 592px;
+}
+
+.work-card h2 {
+  color: #172b4d;
+  font-size: 32px;
+}
+
+.work-card-properties-container {
+  color: #7a869a;
+  font-size: 12px;
+  font-weight: 600;
+}
+
+.work-card-properties-container span {
+  margin-right: 10px;
+}
+
+.work-card-properties-container span:nth-child(1) {
+  font-weight: 700;
+  font-size: 13px;
+  color: #344563;
+}
+
+.work-card p {
+  color: #344563;
+  font-weight: 400;
+  font-size: 15px;
+}
+
+.work-card ul li {
+  display: inline;
+  font-size: 12px;
+  color: #6070ff;
+  margin-right: 10px;
+  height: 40px;
+  padding: 4px 12px;
+  border-radius: 8px;
+  background-color: #ebebff;
+}
+
+.see-project {
+  width: 120px;
+  height: 44px;
+}
+
+.see-project:hover {
+  cursor: pointer;
+  background-color: #396df2;
+  color: #fff;
+}
+
+/* About Section Styles */
+.about {
+  padding: 120px 20px 40px 20px;
+  flex-direction: column;
+  display: flex;
+  width: 100%;
+  height: 1263px;
+  background: #fff;
+  border-top-right-radius: 80px 50px;
+  z-index: 1;
+  position: relative;
+  margin-top: -40px;
+  gap: 10px;
+}
+
+.about h1 {
+  font-size: 40px;
+  width: 50%;
+  color: #172b4d;
+}
+
+.about-details {
+  font-weight: 400;
+  color: #344563;
+}
+
+a.get-my-resume-link {
+  color: #396df2;
+  width: 152px;
+  height: 46px;
+  margin-top: -5px;
+  margin-left: -5px;
+}
+
+a.get-my-resume-link:hover {
+  cursor: pointer;
+  background-color: #396df2;
+  color: #fff;
+}
+
+/* Stack Details Styles */
+.stack-details {
+  padding: 20px;
+  display: flex;
+  flex-direction: column;
+}
```

Comment:
- [ ] As requested by the previous reviewer, I would kindly suggest you work on the alignment of the tech stack and about myself. Currently, it wasn't aligning very well in your design compared to what we have in the Figma template.
  - On line `256` you can change the `padding value` from `20px` to something like `30px 10px` to achieve the proper alignment as what we have in the Figma design. 👍 👍
  
- [ ] Secondly, the previous reviewer stated that you should add the `underline` design to your `Framework` and `Skills` to look exactly like what we have in the Figma design. Doing this will show your ability and confidence to replicate any given design. Please Kindly add it 👍 👍 

- [ ] Also, the previous reviewer requested that you increase your social media icons to 5 in order to look like what we have in the template design. Please kindly fix this.

Your Design 👇 

![ray1](https://user-images.githubusercontent.com/78634841/213477461-b51bf8c1-a67b-4de2-9a99-529982c96ad8.png)

Figma design 👇 

<img src="https://user-images.githubusercontent.com/84156714/212713857-35618f4a-8743-4dd6-b901-0f2f8c6574d5.png">